### PR TITLE
Cole/d3 enhancements

### PR DESCRIPTION
**Pull request by Colelyman**

### CODE REVIEW

**Comment by Colelyman** [comment]
@trevormartinj7 have you had a chance to review this yet? If so, any recommended changes?

File: Makefile
Comment (trevormartinj7) [inline Makefile:1]: Do we want to permanently swap this to max from 4? I think this might be left over from me.

**Review by trevormartinj7 — APPROVED** [review]
Aside from the max vs 4 for CRISPResso large_batch, everything looks good to me! I'll approve, assuming you'll swap it to whatever's best.

File: Makefile
Comment (Colelyman) [reply Makefile:1]: I'm fine keeping it at max, it takes forever...

**Pull request merged**